medusalib/src/main/java/com/trendyol/medusalib/navigator/MultipleStackNavigator.kt

```diff
@@ -229,8 +237,10 @@ open class MultipleStackNavigator(
             val rootFragmentData = FragmentData(rootFragment, createdTag)
             fragmentStackState.notifyStackItemAdd(fragmentStackState.getSelectedTabIndex(), StackItem(createdTag))
             fragmentManagerController.addFragment(rootFragmentData)
+            navigatorListener?.onDestinationChanged(rootFragmentData.fragment)
         } else {
             fragmentManagerController.enableFragment(upperFragmentTag)
+            fragmentManagerController.getFragment(getCurrentFragmentTag())?.let { navigatorListener?.onDestinationChanged(it) }
```

Comment:
This call isn't associated with `loadStackStateFromSavedState`, we need this call for tab changes and back presses, because if i didn't understand wrong `showUpperFragment` is called when user changes the tab or back pressed. @MertNYuksel